### PR TITLE
add sample links.yml

### PR DESCRIPTION
**Pull request by samccann**
##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

docs/docsite/links.yml
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by samccann** [comment]
To be merged after https://github.com/ansible-community/antsibull/pull/355 is merged.

File: docs/docsite/links.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+# This will make sure that plugin and module documentation gets Edit on GitHub links
+# that allow users to directly create a PR for this plugin or module in GitHub's UI.
+# Remove if the collection repository is not on GitHub, or if you do not want this
+# functionality for your collection.
+edit_on_github:
+  repository: ansible-collections/community.REPO_NAME
+  branch: main
+  path_prefix: ''
```
Comment (felixfontein) [inline docs/docsite/links.yml:9]:
```suggestion
  # If your collection root (the directory containing galaxy.yml) does not coincide with your
  # repository's root, you have to specify the path to the collection root here. For example,
  # if the collection root is in a subdirectory ansible_collections/community/REPO_NAME
  # in your repository, you have to set path_prefix to 'ansible_collections/community/REPO_NAME'.
  path_prefix: ''
```

**Review by felixfontein — APPROVED** [review]

File: docs/docsite/links.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+#   1. A link to the issue tracker (if `issues` is specified);
+#   2. A link to the homepage (if `homepage` is specified and does not equal the
+#      `documentation` or `repository` link);
+#   3. A link to the collection's repository (if `repository` is specified).
+
+extra_links:
+  - description: Submit a bug report
+    url: https://github.com/ansible-collections/community.REPO_NAME/issues/new?assignees=&labels=&template=bug_report.md
+  - description: Request a feature
+    url: https://github.com/ansible-collections/community.REPO_NAME/issues/new?assignees=&labels=&template=feature_request.md
```
Comment (gundalow) [inline docs/docsite/links.yml:30]: We don't know what templates (if any) the repo will have, this link should be generic

Is "issue" an OK term for bug or feature?

```suggestion
  - description: Report an issue
    url: https://github.com/ansible-collections/community.REPO_NAME/issues/new/choose
```

File: docs/docsite/links.yml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+      room: '#users:ansible.im'
+  irc_channels:
+    - topic: General usage and support questions
+      network: Libera
+      channel: '#ansible'
+  mailing_lists:
+    - topic: Ansible Project List
+      url: https://groups.google.com/g/ansible-project
+      # You can also add a `subscribe` field with an URI that allows to subscribe
+      # to the mailing list. For lists on https://groups.google.com/ a link is
```
Comment (gundalow) [inline docs/docsite/links.yml:46]:
```suggestion
      # to the mailing list. For lists on https://groups.google.com/ a subscribe link is
```

File: docs/docsite/links.yml [hunk 4]
```diff
@@ -0,0 +1,47 @@
+---
+# This will make sure that plugin and module documentation gets Edit on GitHub links
+# that allow users to directly create a PR for this plugin or module in GitHub's UI.
+# Remove if the collection repository is not on GitHub, or if you do not want this
```
Comment (gundalow) [inline docs/docsite/links.yml:4]:
```suggestion
# Remove this section if the collection repository is not on GitHub, or if you do not want this
```

**Review by gundalow — COMMENTED** [review]
This will be a great improvement to the module docs, thanks.

**Review by gundalow — APPROVED** [review]

**Comment by felixfontein** [comment]
@samccann thanks for adding this!
@gundalow thanks for reviewing!

**Pull request merged**